src/Misc/externals.sh

```diff
@@ -4,17 +4,16 @@ PRECACHE=$2
 LAYOUT_DIR=$3
 L1_MODE=$4
 
-INCLUDE_NODE6=${INCLUDE_NODE6:-true}
+INCLUDE_NODE10=${INCLUDE_NODE10:-true}
 
 CONTAINER_URL=https://vstsagenttools.blob.core.windows.net/tools
 
 NODE_URL=https://nodejs.org/dist
 if [[ "$PACKAGERUNTIME" == "linux-musl-x64" ]]; then
     NODE_URL=https://unofficial-builds.nodejs.org/download/release
-    INCLUDE_NODE6=false
+    INCLUDE_NODE10=false
```

Comment:
This looks  incorrect.  Node6 was excluded b/c it wasn't available in https://unofficial-builds.nodejs.org/download/release.  Doesn't apply to Node10, which is present

I think we can just remove INCLUDE_NODE10